lib/t_web/channels/feed_channel.ex

```diff
@@ -572,13 +570,15 @@ defmodule TWeb.FeedChannel do
     Enum.map(matches, fn
       %Matches.Match{
         id: match_id,
+        inserted_at: inserted_at,
         audio_only: audio_only,
         profile: profile,
         interaction: interaction,
         expiration_date: expiration_date
       } ->
         render_match(%{
           id: match_id,
+          inserted_at: DateTime.from_naive!(inserted_at, "Etc/UTC"),
```

Comment:
I'd pass `match.inserted_at` as is, and ensure it's UTC in the view, that way we cover more area.

Reply:
https://github.com/getsince/test3/pull/492/files